app/repos/exemption.js

```diff
@@ -114,6 +116,22 @@ const handleOrder2023 = (registration, data) => {
   }
 }
 
+/**
+ * @param registration
+ * @param data
+ */
+const setDefaults = (registration, data) => {
+  if (
+    registration.cdo_expiry === null &&
+    data.cdoIssued !== null &&
+    registration._previousDataValues.cdo_issued === null
```

Comment:
I'm a bit nervous about using _previousDataValues as it's a sequelize-specific thing, and I'm not sure they guarantee how it works. Can we not just compare the registration.cdo_issued prior to the call to updateRegistration? It would mean the setDefaults would need to be called before updateRegistration()